ratpack-session-redis/src/main/java/ratpack/session/store/RedisSessionModule.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ratpack.session.store;
+
+import com.google.inject.Scopes;
+import ratpack.guice.ConfigurableModule;
+import ratpack.session.SessionStore;
+
+
+public class RedisSessionModule extends ConfigurableModule<RedisSessionModule.Config> {
```

Comment:
Most of the usage api doc should be here correct as the store itself isn't directly used?